community-website/src/community-project-boilerplate-docgen/src/widgets/search-box.md

```diff
@@ -17,6 +17,7 @@ The Search Box component displays a search box that lets the user search for a s
 | `submitTitle`     | `string`  | The submit button title.
 | `resetTitle`      | `string`  | The reset button title.
 | `searchAsYouType` | `boolean` | If you disable this option, new searches will only be triggered by clicking the search button or by pressing the enter key while the search box is focused.
+| `autofocus?       | `boolean` | Should the search box be focused on render?.
```

Comment:
A backtick is missing after "`autofocus?".
Perhaps the description should have been: "Whether to focus the search box on render."?

Reply:
Sorry @francoischalifour, I merge before I saw your comment. I've copied the text from [react-instantsearch docs](https://community.algolia.com/react-instantsearch/widgets/SearchBox.html#default-props-entry-SearchBox-autoFocus)
Would it make sense to fix the naming everywhere ?

Reply:
It's not a big deal since we're currently moving our docs. If ever you've got time, sure!

Reply:
done here #298 😄